test/pg-cdc/ceased/before-mz-restart.td

```diff
@@ -40,27 +40,5 @@ ALTER TABLE t DROP COLUMN a;
 ALTER TABLE t ADD COLUMN a int;
 INSERT INTO t VALUES (2);
 
-> SELECT true FROM mz_internal.mz_source_statuses WHERE name = 't' AND status = 'ceased' AND error ILIKE 'incompatible schema change%';
-true
-
-$ postgres-execute connection=postgres://postgres:postgres@postgres
-DROP PUBLICATION mz_source;
-INSERT INTO t VALUES (1);
-
-# Primary source detects publication dropped as ceasing error
```

Comment:
just to confirm -- we don't expect the primary source to go into a `ceased` state anymore if this happens?

Reply:
The primary source doesn't currently enter `ceased` at any point because it doesn't encounter a definite error itself. Something we can work on fixing, but this test is accurate as of the behavior on `main`.